### PR TITLE
Docs: HTML embed review.

### PR DESCRIPTION
**Pull request by godai78**
HTML Embed review.

Related features block.

### CODE REVIEW

File: packages/ckeditor5-html-embed/docs/features/html-embed.md [hunk 1]
```diff
@@ -32,6 +32,13 @@ Use the editor below to see the plugin in action.
 
 {@snippet features/html-embed}
 
+## Related features
+
+CKEditor 5 supports a wider range of embed and paste features, including:
```
Comment (Reinmar) [inline packages/ckeditor5-html-embed/docs/features/html-embed.md:37]:
```suggestion
CKEditor 5 supports a wider range of embed and paste features, including:
```

Comment (Reinmar) [inline packages/ckeditor5-html-embed/docs/features/html-embed.md:37]: Rule of thumb – put blank lines the way the content is usually rendered on a page. In this case, after a paragraph, there would normally be a margin separating it from the list.

File: packages/ckeditor5-html-embed/docs/features/html-embed.md [hunk 2]
```diff
@@ -90,22 +97,22 @@ ClassicEditor
 	.catch( ... );
 ```
 
-Currently, the [feature does not execute `<script>` tags](https://github.com/ckeditor/ckeditor5/issues/8326) so content that requires executing JavaScript in order to generate a preview will not show in the editor. However, other JavaScript code – e.g. used in `on*` observers and `scr="javascript:..."` attributes will be executed and therefore a sanitizer still needs to be enabled.
+Currently, the [feature does not execute the `<script>` tags](https://github.com/ckeditor/ckeditor5/issues/8326) so the content that requires executing JavaScript in order to generate a preview will not show in the editor. However, other JavaScript code – e.g. used in `on*` observers and `scr="javascript:..."` attributes will be executed and therefore a sanitizer still needs to be enabled.
```
Comment (Reinmar) [inline packages/ckeditor5-html-embed/docs/features/html-embed.md:100]:
```suggestion
Currently, the [feature does not execute `<script>` tags](https://github.com/ckeditor/ckeditor5/issues/8326) so the content that requires executing JavaScript in order to generate a preview will not show in the editor. However, other JavaScript code – e.g. used in `on*` observers and `scr="javascript:..."` attributes will be executed and therefore a sanitizer still needs to be enabled.
```

These aren't specific script tags. I don't think that "the" is missing here.

File: packages/ckeditor5-html-embed/docs/features/html-embed.md (hunk 1 above)
Comment (godai78) [reply packages/ckeditor5-html-embed/docs/features/html-embed.md:37]: Corrected.

**Comment by AnnaTomanek** [comment]
Added these (and some more) changes directly to `stable` so I'm closing this PR. Thanks for the corrections!

**Pull request closed without merging**